src/Schema/SchemaInterface.php

```diff
@@ -220,6 +220,8 @@ interface SchemaInterface extends ConstraintSchemaInterface
     public const TYPE_JSON = 'json';
     /**
      * Define the abstract column type as `jsonb`.
+     *
+     * @deprecated will be removed in version 2.0.0. Use `SchemaInterface::TYPE_JSON` instead.
      */
     public const TYPE_JSONB = 'jsonb';
```

Comment:
Do we always map JSON type to `jsonb` for PostgreSQL? It actually supports both.

Reply:
Yes, `SchemaInterface::TYPE_JSON` maps to `jsonb` and `jsonb` maps to `SchemaInterface::TYPE_JSON`

https://github.com/yiisoft/db-pgsql/blob/master/src/QueryBuilder.php#L43

https://github.com/yiisoft/db-pgsql/blob/53d7b0e0086dc17dee452ea7c29309ebbdee9ccd/src/Schema.php#L157-L158